src/App.js

```diff
@@ -1,25 +1,19 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+//import FetchWeather from './FetchWeather'; 
+import { Provider } from 'react-redux';
+import Weather from './components/Weather';
+import WeatherStore from './redux/Store'; 
 
-function App() {
+
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <Provider store={WeatherStore}>
+      <div>
```

Comment:
This div isn't needed. It doesn't hurt anything, but it does add another element to the DOM, which can get messy.